docs/api/Server.md

```diff
@@ -20,9 +20,10 @@ The `games` argument takes a list of game implementations
 An object that contains:
 
 1. run (_function_): A function to run the server.
-   Signature: (port, callback) => {}
+   Signature: (port, callback, apiPort) => {}
```

Comment:
I feel like we don't need `port`, `apiPort` and `singlePort`.

How about we make the semantics be:

1. `port` is the only required field.
2. If `apiPort` is not specified, it defaults to the same value that was provided in `port` (i.e. single port mode).
3. If `apiPort` is specified with a different value, then multiple ports are used.

`singlePort` is removed entirely.

Reply:
Sounds good, I'll try!

Reply:
That's fine. I'll document this clearly in the changelog.